src/settings/LocationLocations/LocationManager.js

```diff
@@ -321,6 +325,7 @@ class LocationManager extends React.Component {
     const libraryId = e.target.value;
 
     sessionStorage.setItem(LOCATION_LIBRARY_ID_KEY, libraryId);
+    sessionStorage.setItem(LOCATION_LIBRARY_ID_KEY, '');
```

Comment:
Whoops, I think this belongs in `onChangeCampus` where it would mirror state.

Reply:
Thanks for that! I really missed that.

Reply:
Fixed!